sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1252,6 +1252,10 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 							"requestedOtherPathogenTests, requestedOtherAdditionalTests, 'EXTERNAL', pathogenTestResult, " +
 							"changeDate, creationDate, id, lastOpenedDate, localChangeDate, modified, snapshot, uuid FROM tmp_samples;");
 					getDao(Sample.class).executeRaw("DROP TABLE tmp_samples;");
+				case 168:
+					currentVersion = 168;
+					getDao(Symptoms.class).executeRaw("ALTER TABLE symptoms ADD COLUMN convulsion varchar(255);");
+					getDao(Symptoms.class).executeRaw("ALTER TABLE symptoms ADD COLUMN skinLesion varchar(255);");
```

Comment:
Please make sure that you always update the `currentVersion` variable at the top of the class whenever you add new lines here; it always has to be the last value assigned in this method + 1 (so in this case, 169); also, please make sure to use the correct number here after you've merged the development branch into the pull request

Reply:
"also, please make sure to use the correct number here after you've merged the development branch into the pull request"

I dont understand this part

Reply:
It's just something we noted in one of the last pull requests; you (or Joshua) used 168 (I believe) there even though the version before was 165, i.e. you skipped two versions.

Reply:
Okay, well note.

Reply:
"Please make sure that you always update the `currentVersion` variable at the top of the class"

you mean `DATABASE_VERSION`?

Reply:
@faatihi Yes, absolutely correct!